### PR TITLE
feat(perf-view): Related Transactions button

### PR DESCRIPTION
**Pull request by dashed**
Screen Shot 2020-04-15 at 1 17 03 PM

Add a button on the issue details page that will lead to the peformance transaction summary page.

### CODE REVIEW

**Comment by benvinegar** [comment]
Seems like long term, we should have something in the sidebar or elsewhere that says "this is the related transaction, here is its duration, etc" so that you have a reason to click through.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx [hunk 1]
```diff
@@ -82,6 +86,48 @@ const GroupEventToolbar = createReactClass({
...
+
+    const orgFeatures = new Set(organization.features);
+
+    if (!orgFeatures.has('performance-view')) {
+      return null;
+    }
+
+    const transactionTag = event?.tags?.find(tag => {
+      return tag?.key === 'transaction';
+    });
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx:100]: You don't need to go into tags to get an error's transaction. You can use `event.transaction`.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx [hunk 2]
```diff
@@ -82,6 +86,48 @@ const GroupEventToolbar = createReactClass({
...
+    const to = transactionSummaryRouteWithEventView({
+      orgSlug: orgId,
+      transaction: transactionName,
+      projectID: event.projectID,
+    });
+
+    return (
+      <div key="related-transaction">
+        <RelatedTransactionLink
+          className="btn btn-default btn-sm"
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx:121]: Could we us a Button component here instead of LESS classes.

Comment (dashed) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx]: done in https://github.com/getsentry/sentry/pull/18284/commits/c5c4c40f15fe6d9ecf1672611407b3be0a847851

File: src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx (hunk 1 above)
Comment (dashed) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx:100]: `event.transaction` didn't work; there was no key `transaction` in the `event` object.

Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx:100]: Interesting. It is in many of the events I've looked at.

**Review by markstory — APPROVED** [review]

**Pull request merged**